lib/testing/test-runner.fnl

```diff
@@ -0,0 +1,47 @@
+(local fennel (require :fennel))
+(require :lib.globals)
+(local {: map
+        : slice
+        : pprint}  (require :lib.functional))
+
+(local homedir (os.getenv "HOME"))
+(local customdir (.. homedir "/.spacehammer"))
+(tset fennel :path (.. customdir "/?.fnl;" fennel.path))
+(tset fennel :path (.. customdir "/?/init.fnl;" fennel.path))
```

Comment:
why is this done twice?

Reply:
I didn't want to couple core.fnl and the test-runner.fnl together before I was confident it covers the cases we need.